libs/coin-evm/src/logic.ts

```diff
@@ -27,14 +27,21 @@ export const legacyTransactionHasFees = (tx: EvmTransactionLegacy): boolean =>
 export const eip1559TransactionHasFees = (tx: EvmTransactionEIP1559): boolean =>
   Boolean(tx.type === 2 && tx.maxFeePerGas && tx.maxPriorityFeePerGas);
 
+/**
```

Comment:
Out of the scope of this PR but `logic.ts` means a bit nothing to me. Seems like we only talk about fees in this file could we give it a more relevant name ?

Reply:
Naming could probably be improved, not strong opinion on that.
As of today it looks like this `logic.ts` file is part of the _coin integration framework_ and should hold the "coin specific business logic"

cf. https://github.com/LedgerHQ/developer-portal/pull/773/files#r1255806102

Reply:
Yup agreed, `logic.ts` isn't an amazing name, but that's kinda standard today in the coin integration. 
Naming can surely be improved, but I'm afraid we'll end up with something like `helpers.ts`, and I'm not sure it's really  better 🤷‍♂️